airflow/providers/postgres/hooks/postgres.py

```diff
@@ -89,15 +89,12 @@ def get_conn(self):
         if raw_cursor:
             conn_args['cursor_factory'] = self._get_cursor(raw_cursor)
         # check for ssl parameters in conn.extra
+
         for arg_name, arg_val in conn.extra_dejson.items():
-            if arg_name in [
-                'sslmode',
-                'sslcert',
-                'sslkey',
-                'sslrootcert',
-                'sslcrl',
-                'application_name',
-                'keepalives_idle',
+            if arg_name not in [
+                'iam',
+                'redshift',
+                'cursor',
```

Comment:
I'm not that familiar with psycopg2 but why we exclude those extras?

Reply:
These extras have already been handled elsewhere in the code and do not go as parameters to psycopg2.